libraries/fiff/fiff_stream.cpp

```diff
@@ -2019,9 +2076,7 @@ FiffStream::SPtr FiffStream::start_writing_raw(QIODevice &p_IODevice, const Fiff
 
         t_pStream2 = FiffStream::SPtr();
     }
-    //
-    // Unused parameters from original file (could eventually put these in info struct)
-    //
+
     QList<fiff_int_t> values;
```

Comment:
This code block (line 2080 to 2103) can be removed if you write the tags `FIFF_EXPERIMENTER, FIFF_DESCRIPTION, FIFF_PROJ_ID, FIFF_PROJ_NAME, FIFF_LINE_FREQ, FIFF_XPLOTTER_LAYOUT` to stream later in this function below the [General comment](https://github.com/mne-tools/mne-cpp/blob/04e1c62ceb276521168c981f96853ea163cf7d1c/libraries/fiff/fiff_stream.cpp#L2105).